esdk-performance-testing/benchmarks/python/requirements.txt

```
@@ -0,0 +1,31 @@
+# ESDK Performance Testing - Python Dependencies
+
+# Core dependencies
+pyyaml>=6.0
+psutil>=5.9.0
+numpy>=1.24.0
+pandas>=2.0.0
+matplotlib>=3.7.0
+seaborn>=0.12.0
+
+# Performance measurement
+memory-profiler>=0.61.0
+py-spy>=0.3.14
+
+# Statistics and analysis
+scipy>=1.10.0
+statsmodels>=0.14.0
+
+# Progress and logging
+tqdm>=4.65.0
+colorlog>=6.7.0
+
+# AWS and ESDK dependencies (public packages)
+aws-encryption-sdk>=3.1.0
```

```suggestion
aws-encryption-sdk>=4.0.1
aws-cryptographic-material-providers>=1.11.0
```
You need ESDK >4 to use the MPL (4.0.1 has a nice fix too);
MPL is an optional dependency for ESDK >4, you need to declare it explicitly